### PR TITLE
Feat/cloud formation tag

### PR DESCRIPTION
**Pull request by J3suis**
Add CloudFormationUrl to Report Document for Tagbot

### CODE REVIEW

File: tagging/utils/documents.go [hunk 1]
```diff
@@ -21,12 +21,14 @@ import (
 )
 
 // TaggingReportDocument is an entry in ES' tagging index
+
```
Comment (GabrielRavier) [inline tagging/utils/documents.go:34]: Is this empty line supposed to be there ?

File: tagging/updateTags.go [hunk 2]
```diff
@@ -92,7 +92,9 @@ func UpdateTagsForUser(ctx context.Context, userId int) error {
 			logger.Error(fmt.Sprintf("Generation of tagging reports for resources of type '%s' failed: %s", processor.Name, err.Error()), nil)
 		}
 	}
-
+	for i := range documents {
+		documents[i] = documents[i].GenCloudFormationUrl()
```
Comment (GabrielRavier) [inline tagging/updateTags.go]: Wouldn't it make more sense for this to be written without the assignment (with `GenCloudFormationUrl` taking its argument by pointer) ?

File: tagging/utils/cloudFormationUrl.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"fmt"
+)
+
+const urlFormat = "https://console.aws.amazon.com/cloudformation/home?region=%s#/stacks/stackinfo?stackId=%s"
+const stackId = "aws:cloudformation:stack-id"
+
+func (doc TaggingReportDocument) GenCloudFormationUrl() TaggingReportDocument {
```
Comment (GabrielRavier) [inline tagging/utils/cloudFormationUrl.go]: It seems like it would make more sense to put this function directly in `documents.go`

**Review by GabrielRavier — CHANGES_REQUESTED** [review]

**Review by EthanHusser — COMMENTED** [review]
Nothing to say for this version.

**Review by simonmeyerrr — APPROVED** [review]

**Review by EthanHusser — APPROVED** [review]

**Review by GabrielRavier — APPROVED** [review]
LGTM now

**Pull request merged**